combat-trainer.lic

```diff
@@ -508,11 +530,19 @@ class LootProcess
     @loot_timer = Time.now
   end
 
+  def necro_casting?(game_state)
+    return false unless DRStats.necromancer?
+    return true if game_state.prepare_consume
+    return true if game_state.casting_consume
+    return true if game_state.prepare_cfb
+    return true if game_state.casting_cfb
```

Comment:
this could return false if all of the above conditions aren't met, but 'nil' is falsey.

Reply:
I'm fine with this being explicit.

Does this method need to return `false` (unconditionall) at the very end?